modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -5804,10 +5804,7 @@
                     {
                       "title": "Resource not found",
                       "status": "404",
-                      "detail": "Resource not found",
-                      "source": {
-                        "pointer": "/Users/rockwellrice/Documents/oddball/vets-api/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in `attachments'"
-                      }
+                      "detail": "Resource not found"
                     }
```

Comment:
I think we still need the source and pointer value in there, just shortened down, perhap something like `/modules/claims_api/app/controllers/claims_api/v2/veterans/disability_compensation_controller.rb:74:in ...`

Reply:
Apologies, I think I misinterpreted your original comment; added back and shortened to remove local directory reference in latest push.

Reply:
no worries, I could have been clearer